cg/cli/workflow/taxprofiler/base.py

```diff
@@ -189,3 +190,25 @@ def start(
         use_nextflow=use_nextflow,
         dry_run=dry_run,
     )
+
+
+@taxprofiler.command("start-available")
+@DRY_RUN
+@click.pass_context
+def start_available(context: click.Context, dry_run: bool = False) -> None:
+    """Start full workflow for all cases available for analysis."""
+
+    analysis_api: AnalysisAPI = context.obj.meta_apis[MetaApis.ANALYSIS_API]
+
+    exit_code: int = EXIT_SUCCESS
+    for case in analysis_api.get_cases_to_analyze():
+        try:
+            context.invoke(start, case_id=case.internal_id, dry_run=dry_run)
+        except CgError as error:
+            LOG.error(error)
+            exit_code = EXIT_FAIL
+        except Exception as error:
+            LOG.error(f"Unspecified error occurred: {error}")
+            exit_code = EXIT_FAIL
```

Comment:
```suggestion
        except Exception, CgError as error:
            LOG.error(error)
            exit_code = EXIT_FAIL
```

Reply:
I got an error that this syntax is not valid in python3.9 and I am wondering if I shall leave it as it is.